Assignment4.java

```diff
@@ -0,0 +1,129 @@
+
+package assignment4;
+
+import java.util.Scanner;
+
+public class Assignment4 {
+
+    /**
+     * Curtis Kunkle
+     * Assignment 4
+     * This program uses methods to display the results
+     * of a multiple choice survey.
+     */
+    public static void main(String[] args) {
+        
+        //declare character for user input
+        int choice;
+        
+        //new scanner object!
+        Scanner keyboard = new Scanner(System.in);
+        
+        //Output prompt, list killer Peavey tube amps
```

Comment:
http://peavey.com/products/index.cfm/category/580/Instrument%20Amplifiers

Peavey is an audio equipment company.  They make really solid vacuum-tube driven guitar amplifiers.